### ISSUE TITLE
Allow "channel" in command line for conda info

### ISSUE THREAD

**Issue by tomashek** [conda/conda#5260]
Feature request:

Most of the conda commands allow specification of a channel at the command line with the -c or --channel parameters. Conda info does not, which is inconsistent and makes it more difficult to work with multiple channels and/or labels.

My current workaround is manually adding and then removing the channel from condarc since 'conda info' works on all channels in the config file.

```
$ conda info -c intel intelpython3_core=2017.0.2
usage: conda [-h] [-V] command ...
conda: error: unrecognized arguments: -c
```

**Comment by bkreider** [comment]
Yes.  I meant to log this multiple times over the last N years.  It's difficult to check dependency information without editing the .condarc and then running `conda info` and then reverting .condarc.

**Comment by tyler-thetyrant** [comment]
This is addressed pr #5597 
Starting in conda 4.4.0, you will be able to do
```
> conda search intel::python=3 --info
Loading channels: done

python 3.5.2 intel_0
--------------------
file name   : python-3.5.2-intel_0.tar.bz2
name        : python
version     : 3.5.2
build string: intel_0
build number: 0
size        : 11.7 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.2-intel_0.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip


python 3.5.2 intel_1
--------------------
file name   : python-3.5.2-intel_1.tar.bz2
name        : python
version     : 3.5.2
build string: intel_1
build number: 1
size        : 11.8 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.2-intel_1.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip


python 3.5.2 intel_18
---------------------
file name   : python-3.5.2-intel_18.tar.bz2
name        : python
version     : 3.5.2
build string: intel_18
build number: 18
size        : 11.8 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.2-intel_18.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip


python 3.5.2 intel_19
---------------------
file name   : python-3.5.2-intel_19.tar.bz2
name        : python
version     : 3.5.2
build string: intel_19
build number: 19
size        : 11.8 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.2-intel_19.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip


python 3.5.2 intel_27
---------------------
file name   : python-3.5.2-intel_27.tar.bz2
name        : python
version     : 3.5.2
build string: intel_27
build number: 27
size        : 19.9 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.2-intel_27.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip


python 3.5.3 intel_1
--------------------
file name   : python-3.5.3-intel_1.tar.bz2
name        : python
version     : 3.5.3
build string: intel_1
build number: 1
size        : 24.5 MB
arch        : x86_64
constrains  : ()
platform    : Platform.osx
license     : PSF
subdir      : osx-64
url         : https://conda.anaconda.org/intel/osx-64/python-3.5.3-intel_1.tar.bz2
dependencies:
  - intelpython
  - openssl 1.0.2*
  - sqlite 3.13.*
  - tcl 8.6.4
  - tk 8.6.4
  - xz 5.2.*
  - zlib 1.2.*
  - pip
```

**Comment by tyler-thetyrant** [comment]
Closing since #5597
